tests/tf-manifests/rhcs/machine-pools/rosa-hcp/variable.tf

```diff
@@ -90,4 +90,9 @@ variable "openshift_version" {
 variable "kubelet_configs" {
   type    = string
   default = null
+}
+
+variable "disk_size" {
```

Comment:
Please also update the golang struct for this new variable in utils folder

Reply:
It does already exist as it was used with Classic